scripts/generate_fake_metadata.py

```diff
@@ -0,0 +1,170 @@
+    print(hosts)
+    result = df.to_dict('list')
+    result.update({('Subjects', 'HostSubjectId'): hosts})
+    print(result[('Subjects', 'HostSubjectId')])
+    return result
+
+
+def generate_specimen(df, peep, subject_count, specimen_count):
+    new_cols = {}
+    new_cols[('AdditionalMetaData', 'SubjectIdCol')] =\
+        [subject_count + int(ID) for ID in df['AdditionalMetaData']['SubjectIdCol']]
+
+    print(new_cols[('AdditionalMetaData', 'SubjectIdCol')])
+    new_cols[('Study', 'StudyName')] = [f'{peep}_Study'] * len(df)
+    new_cols[('Specimen', 'SpecimenID')] = [f'Specimen{specimen_count + count}' for count in range(len(df))]
+    new_cols[('Aliquot', 'AliquotID')] =\
+        [f'{spec}-Aliquot{count}' for count, spec in enumerate(new_cols[('Specimen', 'SpecimenID')])]
+    new_cols[('Sample', 'SampleID')] = [f'{spec}-Sample{count}' for count, spec in
+                                        enumerate(new_cols[('Aliquot', 'AliquotID')])]
+    result = df.to_dict('list')
+    result.update(new_cols)
+    return result
+
+
+def generate_aliquot_id_request(peep, specimen_ids):
+    table = defaultdict(list)
+    for _ in specimen_ids:
+        table['StudyName'].append(f'{peep}_Study')
+        # Random Specimen
+        table['SpecimenID'].append(choice(specimen_ids))
+        table['AliquotWeight'].append(float(randrange(1, 9)) / float(randrange(2, 10)))
+    return table
+
+
+def generate_sample_id_request(peep, aliquot_ids):
+    table = defaultdict(list)
+    dates = ['2020-03-01', '2021-02-12', '1999-06-23']
+    versions = ['0.3.0', '1.0.1', '1.3.0']
+    tools = ['Illumina', 'GenBank', 'GreenGenes']
+    conditions = ['Normal', 'Exceptional', 'Subpar']
+    protocol = ['Nothing to report', 'Something to report', 'A lot to report']
+    for ali in aliquot_ids:
+        table['StudyName'].append(f'{peep}_Study')
+        table['SampleProcessor'].append(peep)
+        # Random Aliquot
+        table['AliquotID'].append(choice(aliquot_ids))
+        table['SampleToolVersion'].append(choice(versions))
+        table['SampleTool'].append(choice(tools))
+        table['SampleConditions'].append(choice(conditions))
+        table['SampleDatePerformed'].append(choice(dates))
+        table['SampleProtocolInformation'].append(choice(protocol))
+        table['SampleProtocolID'].append(people.index(peep))
+
+    return table
+
+
+def write_table(table, output_path):
+    lines = ['\t'.join([key for key in table.keys()])]
+    length = len(table['StudyName'])
+    for row in range(length):
+        new_line = []
+        for item in table.values():
+            new_line.append(str(item[row]))
+        lines.append('\t'.join(new_line))
+
+    output = Path(output_path)
+    if not output.exists():
+        output.touch()
+    Path(output_path).write_text('\n'.join(lines) + '\n')
+
+
+def generate():
+    subject_count = 0
+    specimen_count = 0
+    path = Path('/home/david/Work/stash/demo_files/')
```

Comment:
Not a big thing since this is clearly just for the demo, but hard coded paths are always problems waiting to happen.